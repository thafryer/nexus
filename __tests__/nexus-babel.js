import * as babel from '@babel/core'
import Nexus from '../nexus/nexus'

const h1 = `Text('h1','Hey')`
const h2 = `Text('h2','Bye')`

describe('Text', () => {
  it('creates a h1 with Hey', () => {
    const { code } = babel.transform(h1, { plugins: [Nexus] })
    expect(code).toMatchSnapshot()
  })

  it('creates a h2 with Bye', () => {
    const { code } = babel.transform(h2, { plugins: [Nexus] })
    expect(code).toMatchSnapshot()
  })
})
