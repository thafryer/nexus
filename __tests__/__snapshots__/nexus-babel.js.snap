// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text creates a h1 with Hey 1`] = `"<h1>Hey</h1>;"`;

exports[`Text creates a h2 with Bye 1`] = `"<h2>Bye</h2>;"`;
